via/static/styles/video_player.css

```diff
@@ -0,0 +1,7 @@
+/*
+CSS entry point for the video player app.
+*/
+
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
The entry point is CSS rather than SASS for now. We can change this as and when we need SASS features.